### DISCUSSION TITLE
Set Installation Tutorial For .Net 8 Blaozr (ServerSide & WebAssembly)

### DISCUSSION THREAD

**Discussion by MahdiElahi** [vikramlearning/blazorbootstrap#452] [General]
Hi,

Please Take New Tutorial For .Net 8 Blazor Installetion

**Comment by gvreddy04** [comment]
@MahdiElahi Thank you for reaching us. Please see the docs for Web Assembly and Server (Global location).

Blazor WebAssembly (.NET 8): https://docs.blazorbootstrap.com/getting-started/blazor-webassembly-net-8
Blazor Server (.NET 8) - Interactive render mode Server - Global location: https://docs.blazorbootstrap.com/getting-started/blazor-server-net-8
